tst/EnergyPlus/unit/ConvectionCoefficients.unit.cc

```diff
@@ -596,7 +596,7 @@ TEST_F(EnergyPlusFixture, ConvectionCoefficientsTest_EvaluateIntHcModelsFisherPe
     // Test 2: Ceiling Diffuser Model
     ConvModelEquationNum = HcInt_FisherPedersenCeilDiffuserCeiling;
     Hc = 0.0;
-    HcExpectedValue = 6.333;
+    HcExpectedValue = 2.234;
```

Comment:
How did this change so much?  I thought the only diff would be the tiny precision change due to the repeated code.

Reply:
The same functions are called for the zonal level ceiling diffuser algorithm as the surface level algorithm in `EvaluateIntHcModels` except that they were calculating the ACH differently. [The version used in EvaluateIntHcModels](https://github.com/NREL/EnergyPlus/blob/a837bb7430496285acd0d88e081fb7bc3a0fa3f4/src/EnergyPlus/ConvectionCoefficients.cc#L5961-L5970) wasn't applying [the MinFlow constraint used at the zonal level](https://github.com/NREL/EnergyPlus/blob/a837bb7430496285acd0d88e081fb7bc3a0fa3f4/src/EnergyPlus/ConvectionCoefficients.cc#L4325).

The unit tests use a very small flow rate with a 1 m^3 volume which trigger the MinFlow constraint after I created a [common function to calculate ACH](https://github.com/NREL/EnergyPlus/pull/7202/commits/20b7c4c7d126f65f4c240f35c5581d82da25637f#diff-3edd28106d5270bbc5fee24bd22e637bR4265) for use in the Ceiling Diffuser algorithms.

Reply:
Makes sense.  Thanks for the explanation.

Reply:
FWIW I think flow rates this low almost never happen in real models. This change didn't trigger any diffs in our test suite (though not many of them use the ceiling diffuser algorithm).